src/intrprtr.c

```diff
@@ -1183,6 +1183,45 @@ void            IntrQUIT ( void )
     STATE(IntrReturning) = STATUS_QQUIT;
 }
 
+/****************************************************************************
+ **
+ *F  IntrHelp()
+ **
+ **  'IntrHelp' is the action to interpret a help statement.
+ **
+ */
+void IntrHelp(Obj topic)
+{
+    UInt hgvar;
+    Obj  help;
+
+    if (STATE(IntrReturning) > 0) {
+        return;
+    }
+    if (STATE(IntrIgnoring) > 0) {
+        return;
+    }
+    if (STATE(IntrCoding) > 0) {
+        SyntaxError("'?' cannot be used in this context");
+        return;
+    }
+
+    /* FIXME: Hard coded function name */
```

Comment:
How would this "FIXME" ever be resolved, though? Do you want a configurable function name?